core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -49,11 +61,21 @@
 import static io.trino.testing.TestingConnectorSession.SESSION;
 import static io.trino.type.ColorType.COLOR;
 import static java.lang.Float.floatToRawIntBits;
+import static java.lang.Math.max;
+import static java.lang.Math.min;
+import static java.lang.String.format;
+import static java.util.Collections.unmodifiableList;
 import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toUnmodifiableSet;
 import static org.testng.Assert.assertEquals;
 
 public final class BlockAssertions
 {
+    private static final int ENTRY_SIZE = 4;
+    private static final int MAX_STRING_SIZE = 50;
+    private static final Random RANDOM = new Random(633969769);
```

Comment:
Is that a cheat code for some old game? If so which one? If not then I am disappointed.

Reply:
I don't follow. Why are you disappointed?

Reply:
I like magic numbers to have meanings. Like I often use `32167` which gives you 5 black dragons in Heroes of Might and Magic 2.

Reply:
this one actually gives you 98 headshot percentage in CS:GO but let's call it RANDOM_SEED